pkg/sip/signaling.go

```diff
@@ -90,16 +90,6 @@ func sdpMediaOffer(rtpListenerPort int) []*sdp.MediaDescription {
 			Value: styp + " " + codec.Codec.Info().SDPName,
 		})
 	}
-	slices.SortFunc(formats, func(a, b string) int {
```

Comment:
not sorting makes sense, to respect the preference of the offer. do you remember why we sorted to begin with?

Reply:
I think it was a mistake to begin with. The `rtpmap` were always in priority order, I probably forgot to remove sorting here.